### DISCUSSION TITLE
Count on "Notifications" showing wrong numbers.

### DISCUSSION THREAD

**Discussion by PranamBhat** [community/community#3605] [Discussions] [answered]
I have noticed that we have a bug on number of notifications on clicking "Notification" icon on top of GitHub page.

![image](https://user-images.githubusercontent.com/32120033/112433508-5f4e1500-8d68-11eb-99a2-83187bc1966f.png)

Here it says **1-22 of 23** which is wrong.

Secondly, when I mark as "Done" all the notifications, it says **1-0 of 1**. Which is wrong way to show the count!

![image](https://user-images.githubusercontent.com/32120033/112433705-a20fed00-8d68-11eb-9974-37e6b8e70253.png)

**Comment by PranamBhat** [comment]
@alexcnichols  @becca @bailey @evi-liu @michellemerrill Need your help here!

**Reply by fvictorio** [reply]
The same thing is happening to me. @PranamBhat did this eventually go away for you?

**Comment by peterazmanov** [comment]
Moreover when the page I'm viewing isn't the first one `Prev` button doesn't work after checking all visible notifications as `Done`.
![image](https://user-images.githubusercontent.com/15322318/134813986-cc8feb05-8ad4-40d7-8d06-f03270da6847.png)

**Reply by peterazmanov** [reply]
The `Prev` button seems to work now, but the counter is still wrong. I don't understand why the discussion has been resolved, the answer provided is for another related issue...

**Comment by chkoreff** [comment]
Same problem here, it says "1-0 of 1".  Is there any way to bring this to the attention of someone in development?  I don't want to see "1-0 of 1" a year from now.

**Reply by fvictorio** [reply]
And it's not just the `1-0 of 1` that is annoying. The worst part for me is that the blue dot in the notifications icon won't go away even after I've processed all my notifications.

**Comment by wojsmol** [comment]
I have the same problem after reading all notifications from one repository - `actions/runner-images`.

**Reply by radar-blip** [reply]
#6874

**Comment by radar-blip** [comment, marked as answer]
Some of the suggestions here will help:  #6874
Its just a matter of making sure all notifications get marked as Read and/or Done.
Scroll down to about 3/4 down the page.

**Reply by fvictorio** [reply]
Thank you! [This comment](https://github.com/orgs/community/discussions/6874#discussioncomment-1872442) did the trick for me.

**Reply by radar-blip** [reply]
Yep, thats the one that will do it.  👍🏻

**Comment by hannojg** [comment]
I think I have the same:

<img width="1512" alt="Screenshot 2023-06-05 at 12 44 33" src="https://github.com/community/community/assets/16821682/654832aa-25ec-4b44-9f8f-d020d2be1236">


<img width="1512" alt="Screenshot 2023-06-05 at 12 40 17" src="https://github.com/community/community/assets/16821682/9fbae49b-64a5-4b53-b315-9fa9ba5f346e">

**Reply by fvictorio** [reply]
Workaround: https://github.com/orgs/community/discussions/6874#discussioncomment-1872442

**Comment by soredake** [comment]
Same problem for me.
![image](https://github.com/community/community/assets/5204968/ac664206-6bbd-4f22-9b13-4889a6d71244)
